build/NugetProperties.props

```diff
@@ -18,7 +19,7 @@
     <Version>4.0.0</Version>
```

Comment:
When we build official preview release, can we just hard code the version here, for example, `4.0.0-preview` here?

Reply:
This code auto-generates a random string to append to the preview version name, leading to versions such as `3.0.0-preview-010550001-251` and `3.0.0-preview-011100001-1152`. We could hard code the version for official builds and use a different naming convention like `4.0.0-preview1`, `4.0.0-preview2` instead.

Reply:
The random generated numbers were helpful to prevent collisions when we were doing a bunch of dev builds and pushing to our dev NuGet feed. Now that we have a more stable release process and we don't collide, I think it would be great to see the clear version like `4.0.0-preview`

Reply:
The random numbers could still be tacked onto the end for dev builds if we want.

Reply:
What if we want to release multiple preview versions? We can name them as `4.0.0-preview1`, `4.0.0-preview2` etc.

Reply:
We can do `4.0.0-preview` first. If we find we need another preview release later, we can still do `4.0.0-preview2`. At this point, I think we only plan to do one preview release.

Reply:
I don't see anything wrong with starting with `preview1` but I don't see it as necessary either. I would personally just put preview and not assume that we will have a `preview2`, fine either way though.